### PR TITLE
[irtrans] Adapt documentation

### PR DESCRIPTION
**Pull request by lsiepel**
Resolves: https://github.com/openhab/openhab-addons/issues/9532 by improving the markdown.

### CODE REVIEW

File: bundles/org.openhab.binding.irtrans/README.md [hunk 1]
```diff
@@ -16,30 +21,68 @@ There is no specific binding configuration required.
...
+| Parameter | Description                                                                                                                                                                      | Config   | Default |
+|-----------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------|---------|
+| led       | Specifies the led on which infrared commands will be emitted                                                                                                                     | Required | -       |
+| remote    | The remote or manufacturer name which's commands will be allowed, as defined in the IRtrans server database that is flashed into the transceiver (can be ' \*' for 'any' remote) | Required | -       |
+| command   | The name of the command will be allowed, as defined in the IRtrans server database that is flashed into the transceiver (can be '*' for 'any' command)                           | Required | -       |
```
Comment (lolodomo) [inline bundles/org.openhab.binding.irtrans/README.md:83]: I do not find any channel configuration defined for this channel type in XML files. Is it something added dynamically ?

Comment (lsiepel) [reply bundles/org.openhab.binding.irtrans/README.md]: Yes thanks, removed. Somehow i got this dupliced to this channel.

**Review by lolodomo — APPROVED** [review]
LGTM, thank you

**Comment by lsiepel** [comment]
Ignore Jenkins, it’s not related to this PR

**Pull request merged**